Add a view documentation link to the PlacementRule deprecation banner

mprahl opened the pull request:
This varies from the mock up design to conform to other parts of the console.

![image](https://user-images.githubusercontent.com/11711106/231195338-de6551a2-89d6-4b07-b54f-129833172aee.png)

Relates:
https://issues.redhat.com/browse/ACM-4744

mprahl commented:
@dockerymick could you please review this?

mprahl commented:
/cc @dhaiducek @JustinKuli @yiraeChristineKim 

/hold for review from docs

dockerymick commented on frontend/src/components/PlacementRuleDeprecationAlert.tsx:
```diff
@@ -1,15 +1,19 @@
       <Alert variant="info" isInline title={t('Placement rule deprecation')} style={{ marginBottom: '10px' }}>
-        <p>{t('PlacementRule is deprecated and will not receive updates or fixes. Best practice: Use Placement.')}</p>
+        <TextContent>
+          {t('PlacementRule is deprecated and will not receive updates or fixes. Best practice: Use Placement.')}
```
@mprahl 

I suggest the following updates:

Change PlacementRule is deprecated and will not receive updates or fixes.

TO 

Placement rule is deprecated and will not receive updates or fixes.
OR
PlacementRule resource is deprecated and will not receive updates or fixes.

Not sure if we can move Best practice to the next line and bold "Best practice:"

mprahl commented:
@dockerymick could you please provide feedback on the latest iteration?
![image](https://user-images.githubusercontent.com/11711106/231459044-664054b4-7e00-4b4a-91ea-858e3738370e.png)

mprahl commented:
The test failure is unrelated to this PR.

dockerymick commented on frontend/public/locales/en/translation.json:
```diff
@@ -30,6 +30,7 @@
   "< Back to all {{resourceType}} resources": "< Back to all {{resourceType}} resources",
   "<0>Run this command</0> to import your cluster": "<0>Run this command</0> to import your cluster",
   "<0>Run this command</0> to log-in to the existing cluster in your terminal": "<0>Run this command</0> to log-in to the existing cluster in your terminal",
```
Remove the hyphen that is in between log and in (`log-in`). Update to `log in

mprahl commented:
> @mprahl There's a failing test currently. Can you please check if it fails consistently?
> 
> Also - where did you get the translations? In general, you only need to update the English file, and the translation team will update our languages.

Thanks! They were a best guess since it was just splitting up the existing text but I removed it from non-English translation files.

I just rebased and will see if the unrelated test failure is fixed.

/unhold

mprahl commented:
This looks like an issue in this PR. I'll investigate.

yiraeChristineKim commented on frontend/public/locales/ko/translation.json:
```diff
@@ -1895,7 +1895,6 @@
   "Placement rules define the target clusters where subscriptions are delivered. This is done by cluster name, cluster resource annotation(s), or cluster resource label(s).": "배치 규칙은 서브스크립션이 제공되는 대상 클러스터를 정의합니다. 이는 클러스터 이름, 클러스터 리소스 주석 또는 클러스터 리소스 레이블로 수행됩니다.",
   "Placement rules determine which clusters a resources will be applied.": "배치 규칙은 리소스를 적용할 클러스터를 결정합니다.",
   "PlacementRule": "배치 규칙",
-  "PlacementRule is deprecated and will not receive updates or fixes. Best practice: Use Placement.": "배치 규칙은 더 이상 사용되지 않으며 업데이트 또는 수정 사항을 수신하지 않습니다. 모범 사례: 배치를 사용합니다.",
```
who translate this though

mprahl commented:
@KevinFCormier could you please review this again? The last commit explains the CI failure fix.

KevinFCormier commented:
/lgtm
